rules/linux/defense_evasion_busybox_binary.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2022/03/15"
+maturity = "production"
+updated_date = "2022/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary busybox abuse to break out from restricted environments by spawning an interactive system
+shell.This activity is not standard use with this binary for a user or system administrator. It indicates a potentially
+malicious actor attempting to improve the capabilities or stability of their access.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via busybox Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/busybox/"]
+risk_score = 47
+rule_id = "e9b4a3c7-24fc-49fd-a00f-9c938031eef1"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Defense Evasion", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and process.name : "busybox" and process.args_count == 2 and process.args : ("/bin/sh", "/bin/ash", "sh", "ash")
```

Comment:
Maybe do you meant `bash` on this one?

Reply:
Also, is args_count needed here?

Reply:
Its definetly ash :) 
![image](https://user-images.githubusercontent.com/91139415/158427288-ba08d6cc-8508-4652-a2d4-bedba55346d3.png)

Reply:
args_count tightens the query and avoids false positives like below and saves the user from analyzing the alert 
```
{
  "_id": "-cJzjn8BfJgn-82XzsUc",
  "_index": ".ds-logs-endpoint.events.process-default-2022.03.09-000005",
  "_score": "1",
  "_type": "_doc",
  "@timestamp": "2022-03-15T16:40:00.536Z",
  "agent": {
    "id": "c8a71895-37f3-48c2-ba8e-8bf9e844ac1d",
    "type": "endpoint",
    "version": "7.15.1"
  },
  "data_stream": {
    "dataset": "endpoint.events.process",
    "namespace": "default",
    "type": "logs"
  },
  "ecs": {
    "version": "1.11.0"
  },
  "elastic": {
    "agent": {
      "id": "c8a71895-37f3-48c2-ba8e-8bf9e844ac1d"
    }
  },
  "event": {
    "action": "exec",
    "agent_id_status": "verified",
    "category": "process",
    "created": "2022-03-15T16:40:00.536Z",
    "dataset": "endpoint.events.process",
    "id": "MUGdZk4JkGfMCVNH+++K1WBT",
    "ingested": "2022-03-15T16:40:28.000Z",
    "kind": "event",
    "module": "endpoint",
    "sequence": "660514",
    "type": "start"
  },
  "group": {
    "Ext": {
      "real": {
        "id": "1001",
        "name": "shashankks"
      }
    },
    "id": "1001",
    "name": "shashankks"
  },
  "host": {
    "architecture": "x86_64",
    "hostname": "elastic-test-instance-3",
    "id": "57d2b7239218c2b7728faa7cfdf6fd2f",
    "ip": "127.0.0.1,::1,10.128.0.19,fe80::4001:aff:fe80:13",
    "mac": "42:01:0a:80:00:13",
    "name": "elastic-test-instance-3",
    "os": {
      "Ext": {
        "variant": "Debian"
      },
      "family": "debian",
      "full": "Debian 10",
      "kernel": "4.19.0-18-cloud-amd64 #1 SMP Debian 4.19.208-1 (2021-09-29)",
      "name": "Linux",
      "platform": "debian",
      "type": "linux",
      "version": "10"
    }
  },
  "message": "Endpoint process event",
  "process": {
    "args": "busybox,ash,touch",
    "args_count": "3",
    "command_line": "busybox ash touch",
    "entity_id": "YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU1MzktMTMyOTE4MzYwMDAuNTM2MDk5MTAw",
    "executable": "/usr/bin/busybox",
    "Ext": {
      "ancestry": "YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU1MzktMTMyOTE4MzYwMDAuNTMzODIwMzAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU0NTQtMTMyOTE4Mjk5NjYuODI4MTk2MTAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU0NTQtMTMyOTE4Mjk5NjYuNzg1MTU3NzAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU0NTMtMTMyOTE4Mjk5NjYuMTUwNDc3MDAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU0NDctMTMyOTE4Mjk5NjQuMzgzODEzODAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU0NDctMTMyOTE4Mjk5NjQuMjY4ODI2NjAw,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTUxOC0xMzI4ODk0NjA2Ny4w,YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTEtMTMyODg5NDYwNjIuMA=="
    },
    "hash": {
      "md5": "93e29acf3b67fe8847105371d3bd5f80",
      "sha1": "062c7c785e1c87800c6b76d6648cb10e2297f370",
      "sha256": "f030c248efbe4e2337aac9d53726e9222afd81dc802ad04ee1092dc92d1d15f3"
    },
    "name": "busybox",
    "parent": {
      "args": "-bash",
      "args_count": "0",
      "command_line": "-bash",
      "entity_id": "YzhhNzE4OTUtMzdmMy00OGMyLWJhOGUtOGJmOWU4NDRhYzFkLTU1MzktMTMyOTE4MzYwMDAuNTMzODIwMzAw",
      "executable": "/bin/bash",
      "name": "bash",
      "pid": "5454"
    },
    "pid": "5539"
  },
  "user": {
    "Ext": {
      "real": {
        "id": "1000",
        "name": "shashankks"
      }
    },
    "id": "1000",
    "name": "shashankks"
  }
}```